package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.46",
+  "version": "1.1.47",
```

Comment:
I think this will soon have to be bumped up to .49 or something due to competing branches. Should be as easy as merging in lastest and running `yarn version` and then pushing up and also `git push --tags`

Reply:
I'll do that now